### PR TITLE
palette-freemium

### PR DESCRIPTION
**Pull request by Siboney1**
New document explaining freemium

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/645#" title="Author self-approved">Siboney1</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @Siboney1. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/01-introduction/palette-freemium.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+The second month and every month thereafter, the customer is granted a complimentary 25 kilo-Core-hour (kCh) usage to use freely to manage their Kubernetes clusters with Palette. 
+
+There is the choice to continue to stay in the Free Tier indefinitely, so long as you stay under the 25 kCh consumption. Go over the usage limit of 25 kCh, and the clusters remain visible, but convert to Read-Only. Enter a payment and all usage and capabilities are restored.
+
+
+# Kilo-Core-hour Calculations
+
+Usage is calculated as a concept of kilo-Core-hour or kCh. This is a measurement of the number of CPU Cores multiplied by Time. For instance, for a 4-node cluster which has 16 CPU cores each, one day of management will equate to 4 x 16 CPU cores x 24 hours = 1.54 kCh.
+
+
```
Comment (resmaa) [inline content/docs/01-introduction/palette-freemium.md:23]: Palette sends alert messages to users when 25 kCh is crossed
and if more than 5 active clusters are launched further deployments will be restricted.
We can consider including these points as well after confirming. @truegilb plz confirm.

Comment (resmaa) [inline content/docs/01-introduction/palette-freemium.md:23]: @Siboney1 Shall we place this page under "getting started "  -
 @jgautam @truegilb, plz guide

**Review by resmaa — CHANGES_REQUESTED** [review]

File: content/docs/01-introduction/palette-freemium.md (hunk 1 above)
Comment (truegilb) [reply content/docs/01-introduction/palette-freemium.md:23]: Getting started is a good placement. @Siboney1

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/645#pullrequestreview-942652064" title="Approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/645#" title="Author self-approved">Siboney1</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by resmaa** [comment]
closing the PR to change page location

**Pull request closed without merging**